src/components/List/ApplicationsList.tsx

```diff
@@ -64,4 +78,49 @@ const ApplicationsList: React.FC<ApplicationsListProps> = ({
   )
 }
 
+interface ApplicationRowProps {
+  columns: Array<ColumnDetails>
+  application: ApplicationList
+  index: number
+  handleExpansion: (serialNumber: string) => void
+}
+
+const ApplicationRow: React.FC<ApplicationRowProps> = ({
+  columns,
+  application,
+  index,
+  handleExpansion,
+}) => {
+  const { serial } = application
+  return (
+    <Table.Row
+      key={`ApplicationList-application-${serial}`}
+      onClick={() => handleExpansion(serial as string)}
+    >
+      {columns.map(({ headerName, ColumnComponent }) => (
+        <Table.Cell key={`ApplicationList-row-${index}-${headerName}`}>
+          <ColumnComponent application={application} />
+        </Table.Cell>
+      ))}
+      <Table.Cell icon="angle down" collapsing />
```

Comment:
Any chance you can add an extra column (or maybe just a `colSpan` in the last cell) to the header row so the grey header background and frame extends right to edge to allow for the extra "drop-down" icon?
<img width="301" alt="Screen Shot 2021-02-25 at 12 32 07 AM" src="https://user-images.githubusercontent.com/5456533/108994763-10797500-7701-11eb-9e7e-530f3147e6e9.png">